azurelinuxagent/distro/default/rdma.py

```diff
@@ -21,10 +21,14 @@
 
 import os
 import re
-import azurelinuxagent.logger as logger
+import azurelinuxagent.common.logger as logger
+import azurelinuxagent.common.utils.shellutil as shellutil
 
 
 class RDMAHandler(object):
+    def __init__(self):
+        self.driver_module_name = 'hv_network_direct'
```

Comment:
OK after refreshing my memory a bit I agree that a class member is a nicer choice for the driver_module name